src/google-classroom-extractor/google_classroom_extractor/csv_generation/write.py

```diff
@@ -14,6 +14,7 @@
 SECTION_ASSOCIATIONS_ROOT_DIRECTORY = "data/ed-fi-udm-lms/section={id}/section-associations/"
 ASSIGNMENT_ROOT_DIRECTORY = "data/ed-fi-udm-lms/section={id}/assignments/"
 SUBMISSION_ROOT_DIRECTORY = "data/ed-fi-udm-lms/section={id1}/assignment={id2}/submissions/"
+USER_ACTIVITY_ROOT_DIRECTORY = "data/ed-fi-udm-lms/section={id}/user-activities/"
```

Comment:
In the Schoology extractor, we're note hard-coding `data/ed-fi-udm-lms` - instead handling this "base directory" as configuration parameter. Let's leave it alone for now, and perhaps we can change this in [FIZZ-62](https://tracker.ed-fi.org/browse/FIZZ-62)?

Reply:
In fact, I have a module just for determining the file paths. Perhaps that module should be extracted to a package that can be shared with all of the extractors.

Reply:
That sounds like a good idea.